libs/api/src/lib/graphql/peer.ts

```diff
@@ -20,6 +20,7 @@ export const GraphQLPeerProfileInput = new GraphQLInputObjectType({
   fields: {
     name: {type: new GraphQLNonNull(GraphQLString)},
     logoID: {type: GraphQLString},
+    squareLogoId: {type: GraphQLString},
```

Comment:
`squareLogoId` is nullable in the GraphQL API but is required in `PeerProfileEditPanel.tsx`. As far as I can tell, they both could be non-nullable?

Reply:
we want user to have to set them, but it has to be nullable in graphql as its a new field.

All the other fields are the same